index.js

```diff
@@ -4,106 +4,62 @@ const AWS = require("aws-sdk");
 AWS.config.update({region: 'us-east-1'});
 const ssm = new AWS.SSM();
 const obj = require("./params.json");
-const args = require('minimist')(process.argv.slice(2));
+const args = require('yargs/yargs')(process.argv.slice(2))
+.array('name').array('stage').argv;
```

Comment:
I think it's something like this to do what you wanted in yargs if you wanted alias and some information when using the help command

```suggestion
const args = require('yargs/yargs')(process.argv.slice(2))
   .option('n', {
        alias: 'name',
        describe: 'Names of configurations to deploy',
        type: 'array'
    })
   .option('s', {
        alias: 'stage',
        describe: 'stages of selected configurations to deploy',
        type: 'array'
    }) 
    .argv
```